.github/workflows/ci.yml

```diff
@@ -61,3 +61,7 @@ jobs:
         run: npm install @commitlint/config-conventional
       - name: Validate current commit (last commit) with commitlint
         run: npx commitlint --from HEAD~1 --to HEAD --verbose
+      - name: Install dotnet sdk
+        run: sudo apt install -y dotnet6
+      - name: Run script to make sure all files end with EOL 
+        run: dotnet fsi eofconvention.fsx
```

Comment:
we use camelCase for script names, so can you rename eofconvention to eofConvention?

Reply:
Done